DaysOfCode/dataTypes.js

```diff
@@ -8,41 +8,36 @@ let inputStdinArray = '';
 let inputCurrentline = 0;
 
 process.stdin.on('data', (data) => {
-    inputStdin += data;
+  inputStdin += data;
 });
 
 process.stdin.on('end', () => {
-    inputStdinArray = inputStdin.split('\n');
-    main();
+  inputStdinArray = inputStdin.split('\n');
+  main();
 });
 
 // Reads complete line from STDIN
 function readLine() {
-    return inputStdinArray[inputCurrentline += 1];
+  return inputStdinArray[inputCurrentline += 1];
 }
 
 function main() {
-    const i = 4;
-    const d = 4.0;
-    const s = 'HackerRank';
+  const firtsValue = 4;
+  const secValue = 4.0;
+  const thirdValue = 'HackerRank ';
 
-    // Declare second integer, double, and String variables.
-    let secI;
-    let secD;
-    let secS;
+  // Read and save an integer, double, and String to your variables.
+  const firtsReadLine = parseInt(readLine());
```

Comment:
is this code generated by hacker rank?

Reply:
yes, this code:
```
  var i = 4
  var d = 4.0
  var s = "HackerRank "

// Read and save an integer, double, and String to your variables.
  const firtsReadLine = parseInt(readLine());
  const secReadLine = parseFloat(readLine());
  const thirdReadLine = readLine();

```
but my repository use this code:

```
  const firtsValue = 4;
  const secValue = 4.0;
  const thirdValue = 'HackerRank ';

  // Read and save an integer, double, and String to your variables.
  const firtsReadLine = parseInt(readLine());
  const secReadLine = parseFloat(readLine());
  const thirdReadLine = readLine();
```